tests/units/anta_tests/__init__.py

```diff
@@ -3,10 +3,43 @@
 # that can be found in the LICENSE file.
 """Tests for anta.tests module."""
 
+from __future__ import annotations
+
 import asyncio
-from typing import Any
+from typing import TYPE_CHECKING, Any, Literal, TypedDict
+
+if TYPE_CHECKING:
+    from typing_extensions import NotRequired  # NOTE: required to support Python < 3.11 https://peps.python.org/pep-0655/#usage-in-python-3-11
+
+    from anta.device import AntaDevice
+    from anta.models import AntaTest
+
+
+class AtomicResult(TypedDict):
+    """Expected atomic result of a unit test of an AntaTest subclass."""
+
+    result: Literal["success", "failure", "skipped"]  # TODO: Refactor tests and use AntaTestStatus
```

Comment:
this can be given to one of the GSLabs engineer if clearly defined in an issue

Reply:
https://github.com/aristanetworks/anta/issues/995